pkgs/development/ocaml-modules/lambdasoup/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, fetchFromGitHub, buildDunePackage, markup }:
+
+buildDunePackage rec {
+  pname = "lambdasoup";
+  version = "0.6.3";
+
+  src = fetchFromGitHub {
+    owner = "aantron";
+    repo = pname;
+    rev = version;
+    sha256 = "1w4zp3vswijzvrx0c3fv269ncqwnvvrzc46629nnwm9shwv07vmv";
+  };
+
+  propagatedBuildInputs = [ markup ];
+
+  meta = {
+    description = "Functional HTML scraping and rewriting with CSS in OCaml";
+    homepage = "https://aantron.github.io/lambdasoup/";
+    license = lib.licenses.mit;
```

Comment:
This version is in fact licensed under bsd. Let's fix this up and perhaps leave a comment near the version to remember to switch to MIT?

Reply:
Done. Thanks.